tests/test_generators/python/test_python_ifabsent_processor.py

```diff
@@ -403,9 +403,13 @@ def test_enum_default_value():
             schema_view.all_slots()[SlotDefinitionName("presence")],
             schema_view.all_classes()[ClassDefinitionName("Student")],
         )
-        == "PresenceEnum.Missing"
+        == "field(default=None)"
     )
 
+    print(processor.enum_forward_references)
```

Comment:
no print statements plz :)

Reply:
Ups, late-hours slip. Will remove it. Thanks for the good catch!